src/bin/cargo-flamegraph.rs

```diff
@@ -183,6 +198,13 @@ fn workload(opt: &Opt, artifacts: &[Artifact]) -> anyhow::Result<Vec<String>> {
             unit_test: Some(Some(t)),
             ..
         } => (&["lib", "bin"], t),
+        Opt {
+            unit_bench: Some(Some(t)),
+            ..
+        } => {
+            trailing_arguments.push("--bench".to_string());
```

Comment:
What is this trying to achieve? It seems like a weird setup to append to a copy of `trailing_arguments`.

Reply:
One runs benchmarks with the `--bench` switch provided to the unit test binary. See `<binary> --help`. We were already doing a copy a few lines down, so this doesn't really change much.